near-sdk/src/types/primitives.rs

```diff
@@ -0,0 +1,45 @@
+use near_primitives_core::hash::CryptoHash;
+
+pub use near_primitives_core::runtime::fees::RuntimeFeesConfig;
+
+//* Type aliases from near_primitives_core
+
+/// Account identifier. Provides access to user's state.
+pub type AccountId = String;
+/// Hash used by a struct implementing the Merkle tree.
+pub type MerkleHash = CryptoHash;
```

Comment:
So this is going to be a tough one. SDK defines a simple CryptoHash

![image](https://user-images.githubusercontent.com/1711539/117982167-67750a80-b33e-11eb-979c-d2027ceb28c2.png)

But this MerkleHash is also public :-(

Can we quickly grep existing contracts to see if they use `MerkleHash`? Just to understand what leeway we have with changing that? 

Although it seems that porting primitives-code `CryptoHash` and `Digest` won't be hard, just annoying.

Reply:
Yeah, I don't see a need for it to be exposed, but I was just trying to keep the API exactly as it was for now. I'll try to look to find any examples of using `MerkleHash`, but I doubt there would be any references to it through the SDK.

Reply:
>but I doubt there would be any references to it through the SDK.

I meant grepping *extenral* contracts, like near/core-contracts.

Reply:
Also headsup here -- CrypoHash definition in nearcore master branch is slightly different from the one used here.

Reply:
> > but I doubt there would be any references to it through the SDK.
> 
> I meant grepping _extenral_ contracts, like near/core-contracts.

Yeah, what I meant by that is that I doubt people reference that type through the SDK because there is no connection with the SDK.

> Also headsup here -- CrypoHash definition in nearcore master branch is slightly different from the one used here.

Yeah this is an overridden type, there are a few of these, and I'm not quite sure why yet

Reply:
I don't think `MerkleHash` is used anywhere by the contracts. At least I'm not aware of one